airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2096,6 +2102,13 @@ components:
             - days
             - weeks
             - months
+    NamespaceDefinitionType:
```

Comment:
discussed offline: we should leave a comment here explaining the default behaviour